docs/admin/install.rst

```diff
@@ -117,19 +115,42 @@ Follow the `installation documentation <https://www.qubes-os.org/doc/installatio
 - Use all available storage space for the installation (as the computer should be dedicated to SecureDrop Workstation).
 - Set a strong FDE passphrase - a 6-word Diceware passphrase is recommended.
 - Create an administrative account named ``user`` with a strong password.
+- In the template configuration, ensure that the following options are checked:
+
+  - "Create default system qubes (sys-net, sys-firewall, default DispVM)"
+  - "Make sys-firewall and sys-usb disposable"
+- If there is a grayed out option "USB qube configuration disabled", make a note of this. An additional setup step will be required (see below).
 
-  .. note:: Qubes is not intended to have multiple user accounts, so this account name and password will be shared by all SecureDrop Workstation users. The password will be required to log in and unlock the screen during sessions - choosing something strong but memorable and easily typed is recommended!
+  .. note:: Qubes is not intended to have multiple user accounts, so your account name and password will be shared by all SecureDrop Workstation users. The password will be required to log in and unlock the screen during sessions - choosing something strong but memorable and easily typed is recommended!
 
 Once the installation is complete, you will be prompted to reboot into Qubes. Reboot, removing the install USB when the computer restarts.
 
 You will be prompted to enter the FDE passphrase set during installation.
 
-.. note:: On first booting into a Qubes OS 4.0.4 installation, you may be prompted to enter the FDE passphrase at a command-line prompt rather than via the GUI. The next system update will restore the GUI prompt.
-
 After the disk is unlocked and Qubes starts, you will be prompted to complete the initial setup. Click the Qubes OS icon, then accept the default options and click **Done**. Finally, click **Finish Configuration** to set up the default system TemplateVMs and AppVMs.
 
 Once the initial setup is complete, the login dialog will be displayed. Log in using the username and password set during installation.
 
+If, during the installation, you encountered the grayed out option "USB qube configuration disabled", you must now create a VM to access your USB devices. To do so, open a ``dom0`` terminal via the Qubes menu (the **Q** icon in the upper left corner): **Q > Terminal Emulator**. Run the following command:
+
+.. code-block:: sh
+
+  sudo qubesctl state.sls qvm.sys-usb
```

Comment:
@creviera I've copied this command from https://github.com/freedomofpress/securedrop-workstation/issues/769#issuecomment-1116443142 but have not tested it. If I read the later comments correctly, we may also want to add a hint for USB keyboard users to run `sudo qubesctl state.sls qvm.usb-keyboard`?

Reply:
My workaround in https://github.com/freedomofpress/securedrop-workstation/issues/769#issuecomment-1176793702 is:

> Just noting my workaround is to run sudo qubesctl state.sls qvm.sys-usb which uses the correct disposable template (for 4.1.1-rc this is fedora-26-dvm). And I also have to run sudo qubesctl state.sls qvm.usb-keyboard to fix Denied qubes.InputKeyboard from sys-usb to dom0 errors.
> 
> Reboot seems to be necessary for thumb drives to be show up in the Devices menu under the USB Devices section and to be able to attach drives to VMs.

Both steps were required for me to get around the issue. There may be a way to fix the mapping of internal hardware so that we don't have to run the keyboard salt state. Let me know if you'd like me to investigate next Monday (or potentially this Friday).

Reply:
OK, I actually see those  `Denied`s on my daily machine as well, on plugging in a Yubikey (haven't tried with just a usb stick). So the docs should include both.

Reply:
I've added this additional step, plus a verification step for USB support, in 69ec8ebfa4fd1f8b9cd43c7705ab3f3321c6be7f. (I've not further verified these commands.)